src/containers/Experiment/index.js

```diff
@@ -135,11 +135,37 @@ let Experiment = ({
                 <section className="experiment__section">
                   <h2 className="experiment__title">Samples</h2>
                   <SamplesTable
-                    accessionCode={experiment.accession_code}
-                    samples={experiment.samples}
-                    addSamplesToDataset={addExperiment}
-                    removeSamplesFromDataset={removeSamplesFromExperiment}
-                    dataSet={dataSet}
+                    sampleIds={experiment.samples.map(x => x.id)}
+                    // Render prop for the button that adds the samples to the dataset
+                    pageActionComponent={samplesDisplayed =>
```

Comment:
I like this approach! But could we could we actually make a functional component like `SampleTableActions` and pass that in here? Might make it less verbose

Reply:
Yes! Good idea!